src/containers/ArticleIndex/ArticleIndex.jsx

```diff
@@ -1,14 +1,49 @@
-import React from 'react'
-import ArticleIndexHeading from '../../components/ArticleIndexHeading/ArticleIndexHeading'
+import { React, useState } from 'react'
+import ArticleIndexHeading from '../../components/ArticleIndexHeading/ArticleIndexHeading';
+import Navigation from '../../components/Navigation/Navigation';
+import { useParams } from 'react-router-dom';
+import BackButton from '../../components/BackButton/BackButton';
+import SaveButton from '../../components/SaveButton/SaveButton';
+import "./ArticleIndex.scss";
+// path: /articles/:articleId
 
 
-// path: /articles/:articleId
+const ArticleIndex = ({ articleArray }) => {
+  const { articleId } = useParams();
+
+  const pickedArticle = articleArray.find(({ id }) => id === parseInt(articleId));
+
+  const [isPressed, setIsPressed] = useState(pickedArticle.isActive);
+
+  const onSaveButtonPress = () => {
+    setIsPressed(!isPressed);
+  };
 
-const ArticleIndex = ({articleArray}) => {
   return (
-    <div>
-      <ArticleIndexHeading articleArray={articleArray}/>
+    <div className='articleIndex'>
+      <header className="articleIndex__header">
+        <div className="articleIndex__backButton">
+          <BackButton linkTo="linkTo" />
```

Comment:
What does the back button link to? Would this not be the articles list container?